packages/-serializer-encapsulation-test-app/tests/integration/delete-record-test.js

```diff
@@ -0,0 +1,251 @@
+import Model, { attr } from '@ember-data/model';
+import Store from 'serializer-encapsulation-test-app/services/store';
+import JSONAPIAdapter from '@ember-data/adapter/json-api';
+import EmberObject from '@ember/object';
+import { setupTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { resolve } from 'rsvp';
+
+class Person extends Model {
+  @attr
+  firstName;
+
+  @attr
+  lastName;
+
+  // override to not call serialize()
+  toJSON() {
+    const { id, firstName, lastName } = this;
+    return {
+      id,
+      type: this.constructor.modelName,
+      attributes: {
+        firstName,
+        lastName,
+      },
+    };
+  }
+}
+
+module('inteegration/delete-record - running deleteRecord with minimum serializer', function(hooks) {
```

Comment:
```suggestion
module('integration/delete-record - running deleteRecord with minimum serializer', function(hooks) {
```

Reply:
I've fixed this spelling error in most the already merged tests, suspect you are copying from one we haven't fixed yet